### DISCUSSION TITLE
Shovel question

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#7085] [Q&A] [answered]
Can Shovel mode pick up bales? it seems to  not like that.

Pallets too if possible

**Comment by Tensuko** [comment, marked as answer]
For Bales, was an Idea, but won't happen in the near feature.
Pallets will never happen for some reason (they behave like sh!t)

**Reply by rento112** [reply]
Perhaps you can try taking a look at the CSZ pack fork lifts. They're  able to not only disable  their collision box to go right through the pallet, they can attachh the pallet like an implement.  If it's possible to make courseplay work withh attaching implements with a way point perhaps, you'll give your fans the ability to use forklifts.

**Reply by Tensuko** [reply]
Wont use those autoload stuff. Only Vanilla as Baseversion.
